bitrise.yml

```diff
@@ -11,6 +11,15 @@ workflows:
   ci:
     steps:
     - script@1.1.3:
+        title: Download fastlane...
+        run_if: '{{enveq "GIT_REPOSITORY_URL" "git@github.com:Wolox/ios-base-project.git" }}'
```

Comment:
A question, if I'm starting a new project, copied from the BaseProject and run bootstrap, with this if it wouldn't download the fastlane scripts? And it should.

Reply:
Maybe the script can download the fastlane files if there isn't a fastlane folder yet.

Reply:
Yes, in this case the person that does the kickoff shouldn't worry about doing anything. I can delete what I had added in `tech-guides`.

Reply:
I'm guessing the fastlane mobile project will eventually be a dependency right? So we don't have to copy them each time we kickoff a project.

I'd ideally move the fastlane setup into the bootstrap scripts (idk if that's what dani suggested, I didn't get if she referred to the scrips in the ci or our scripts) until the dependency works. But if this is a really short-term solution I wouldn't waste my time changing it.

Reply:
I'd ideally make fastlane mobile a dependency to install via carthage. Until that, if this solution is really short-term I wouldn't waste my time changing it as long as the kickoff guide explains it. Otherwise I'd move the fastlane mobile setup to the bootstrap scrips (I think that's what dani suggested if I understood correctly)

Reply:
Yes, I though again about this and I think the best option is to leave it this way, and change the kickoff guide as done.

I don't want to edit the scripts with this particular change, or we run the risk of lose it in case we update them.

Reply:
Also, as @damfinkel says, this is a short term solution. We will create an interactive kickoff which does this for us.